### PR TITLE
add nginx IP ACLs

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Background Information
+======================
+
+IP based ACLs can be externally used to allow access (whitelist strategy) to a
+specific web service only by customers so you can easily get rid of most of the
+malicious traffic to the application server.
+This also has some downsides like the site is probably
+not visible to search engines and will not be indexed.
+On the other hand you can also blacklist (blacklist strategy) some bot IPs,
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
On the other hand you can also blacklist (blacklist strategy) some bot IPs and
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    as the only protection for your webservices.
+
+
+Background Information
+======================
+
+IP based ACLs can be externally used to allow access (whitelist strategy) to a
+specific web service only by customers so you can easily get rid of most of the
+malicious traffic to the application server.
+This also has some downsides like the site is probably
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
This also has some downsides: for example, the site will probably
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+not visible to search engines and will not be indexed.
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
not be visible to search engines and will therefore not be indexed.
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+.. image:: images/nginx_ip_acl_01_list_view.png
+
+Click the + button to create a new ACL.
+
+.. image:: images/nginx_ip_acl_02_create_acl_view.png
+
+Next enter a reasonable title, for example here "Allow Private IPs" was used.
+Now the different IP addresses or IP ranges can be entered. In this case some
+common private IP ranges were allowd and the default rule was set to block.
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
common private IP ranges were allowed and the default rule was set to block.
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+A new line can be added by clicking the + icon while trash can deletes the row.
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
A new line can be added by clicking the + icon while the trash can icon deletes the row.
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+This means that this searvice should be only visible internally.
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
This means that this service should be only visible internally.
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+.. Warning::
+    Keep in mind that carrier grade NAT (CGN) may cause some trouble with this
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
    Keep in mind that carrier grade NAT (CGN) may cause some trouble with these
```

File: source/manual/how-tos/nginx_ip_acl.rst [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    ACLs too. Please check how your traffic is handled first.
+
+
+Assign it to a Location, HTTP- or Stream-Server
```
Comment (MichaelDeciso) [inline source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
Assign it to a Location, HTTP or Stream-Server
```

File: source/manual/how-tos/nginx_ip_acl.rst (hunk 2 above)
Comment (fabianfrz) [reply source/manual/how-tos/nginx_ip_acl.rst]: after a column a capital should follow, otherwise I would accept your suggestion

Comment (fabianfrz) [reply source/manual/how-tos/nginx_ip_acl.rst]:
```suggestion
This also has some downsides: For example, the site will probably
```

Comment (MichaelDeciso) [reply source/manual/how-tos/nginx_ip_acl.rst]: No, it should not. Not in English, at least.

Comment (fabianfrz) [reply source/manual/how-tos/nginx_ip_acl.rst]: https://www.grammarly.com/blog/capitalization-after-colons/

-> 3rd heading

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**